src/test/java/com/googleinterns/zoomtube/servlets/LectureServletTest.java

```diff
@@ -94,7 +94,7 @@ public void doPost_urlAlreadyInDatabase_shouldReturnLecture() throws IOException
     servlet.doPost(request, response);
 
     assertThat(datastoreService.prepare(new Query(LectureUtil.KIND)).countEntities()).isEqualTo(1);
-    verify(response).sendRedirect("/view/?id=1&video-id=wXhTHyIgQ_U");
+    verify(response).sendRedirect("/view/?id=1");
```

Comment:
```suggestion
    verify(response).sendRedirect("/view?id=1");
```
I don't think there should be an extra slash after `view`. I think the issue may be the `REDIRECT_URL` in `LectureServlet`.

Reply:
fixing in a separate pr

Reply:
Technically I think it should work with or without the slash, but yeah we should be consistent and not use one.